lib/core/datatable/data/data-table.schema.ts

```diff
@@ -24,7 +24,7 @@ import { ObjectDataColumn } from './object-datacolumn.model';
 
 @Directive()
 // eslint-disable-next-line @angular-eslint/directive-class-suffix
-export abstract class DataTableSchema {
+export abstract class DataTableSchema<CustomColumnData = unknown> {
```

Comment:
should better be `T` here, also can you check if this is not a breaking change for existing apps?

Reply:
As discussed, updated to `T`.